agent/api/task.go

```diff
@@ -674,3 +678,18 @@ func (t *Task) String() string {
 	}
 	return res + "]"
 }
+
+func (task *Task) GetID() (string, error) {
+	parsedARN, err := arn.Parse(task.Arn)
+	if err != nil {
+		return "", errors.Wrapf(err, "task get-id: unable to parse taskARN")
+	}
+
+	resource := parsedARN.Resource
+	resourceSplit := strings.SplitN(resource, arnResourceDelimiter, arnResourceSections)
```

Comment:
Isn't Split enough here instead of SplitN, since taskARN has only single '/'? In that case, `arnResourceSections` may not be necessary.